files/zh-cn/web/javascript/reference/operators/bitwise_xor/index.md

```diff
@@ -5,7 +5,7 @@ slug: Web/JavaScript/Reference/Operators/Bitwise_XOR
 
 {{jsSidebar("Operators")}}
 
-**按位异或**（**`^`**）运算符在其中任意一个操作数对应的二进制位为 `1` 时，该位的结果值为 `1`。
+**按位异或**（**`^`**）运算符在其中任意一个操作数对应的二进制位不同时，该位的结果值为 `1`。
```

Comment:
two suggestions:

## translation of [english doc](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Bitwise_XOR)

```suggestion
**按位异或**（**`^`**）运算符在其中任意一个操作数对应的二进制位为 `1`（但不是两个操作数的对应二进制位均为 `1`）时，该位的结果值为 `1`。
```

## improvement

```suggestion
**按位异或**（**`^`**）运算符在两个操作数对应的二进制位不同时，该位的结果值为 `1`。
```

Reply:
Another version that is in line with English version:

```suggestion
**按位异或**（**`^`**）运算符在两个操作数对应位有且仅有一个为 `1` 时，该位的结果值为 `1`。
```